PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/component/TreeUtils.java

```diff
@@ -163,15 +171,49 @@ private static IntellijAzureActionManager.ActionGroupWrapper toIntellijActionGro
         return new IntellijAzureActionManager.ActionGroupWrapper((ActionGroup) actions);
     }
 
-    public static void renderMyTreeNode(@Nonnull Tree.TreeNode<?> node, @Nonnull SimpleColoredComponent renderer) {
+    public static void renderMyTreeNode(JTree tree, @Nonnull Tree.TreeNode<?> node, boolean selected, @Nonnull SimpleColoredComponent renderer) {
         final NodeView view = node.inner.view();
         if (BooleanUtils.isFalse(node.loaded)) {
             renderer.setIcon(AnimatedIcon.Default.INSTANCE);
         } else {
             renderer.setIcon(IntelliJAzureIcons.getIcon(view.getIcon()));
         }
+        final Object highlighted = tree.getClientProperty(HIGHLIGHTED_RESOURCE_KEY);
+        final boolean toHighlightThisNode = Optional.ofNullable(highlighted).map(h -> ((Pair<Object, Long>) h))
+            .filter(h -> System.currentTimeMillis() - h.getRight() < 10000)
+            .filter(h -> Objects.equals(node.getUserObject(), h.getLeft())).isPresent();
+        if (selected && toHighlightThisNode) {
+            renderer.setBorder(BorderFactory.createLineBorder(JBColor.RED));
+            renderer.setBackground(JBColor.YELLOW);
+            renderer.setForeground(JBColor.RED);
+            renderer.setOpaque(false);
+        } else {
+            renderer.setOpaque(true);
+            renderer.setBorder(null);
+            renderer.setBackground(null);
+            renderer.setForeground(null);
+        }
         renderer.append(view.getLabel(), view.isEnabled() ? SimpleTextAttributes.REGULAR_ATTRIBUTES : SimpleTextAttributes.GRAY_ATTRIBUTES);
         renderer.append(Optional.ofNullable(view.getDescription()).map(d -> " " + d).orElse(""), SimpleTextAttributes.GRAY_ITALIC_ATTRIBUTES, true);
         renderer.setToolTipText(Optional.ofNullable(view.getTips()).orElse(view.getLabel()));
     }
+
+    public static boolean isInAppCentricView(@Nonnull DefaultMutableTreeNode node) {
+        return isInAppCentricView(new TreePath(node.getPath()));
+    }
+
+    public static boolean isInAppCentricView(@Nonnull TreePath path) {
+        if (path.getPathCount() < 2) {
```

Comment:
Why path count < 2 is not app centric view? e.g., the root node of app centric (Resoruce Groups Node) ?

Reply:
root node is `empty`, Resource Groups node is actually not a root node.